### DISCUSSION TITLE
[Feature Request] Add "persistence" toggle to bookmarks' folders switching.

### DISCUSSION THREAD

**Discussion by gety9** [plateaukao/einkbro#325] [Ideas]
Just realized how useful Eink Bro bookmark folders can be! (better late then never :))

It would be great if there will be check box making it remember switching to another folder.

Example:
User has 2 bookmark folders
[TOP] here all regular bookmarks
[Math] here all bookmarks related to his math classes

User sits down to do his math home work, usually it takes him 2 hours, during this 2 hours he needs math bookmarks on average 10 times.

he needs to look up some saved math function
1, open eink bro
2, press bookmarks icon
3, press Math Folder
4, pres bookmark math 1

In 7 min, he needs to look something else (of course math related cause it's math homework time)
1, open eink bro
2, press bookmarks icon
3, press Math Folder
4, pres bookmark math 2

....
You got the point, every time there is this (3, press Math Folder).

If instead he could've checked checkbox "remember bookmark folder switch"
OR pressed button "make current folder top"

so that next time it will go
1, open eink bro
2, press bookmarks icon
3, pres bookmark math3

And after math class he could uncheck the checkbox OR make TOP folder top again.

**Comment by plateaukao** [comment]
will consider it in the future.

**Comment by gety9** [comment]
closing, https://github.com/plateaukao/einkbro/discussions/461 is better idea to spend time on (if decided)
